src/pages/docs/creators/ray/supported-versions-and-other-limitations.md

```diff
@@ -11,7 +11,7 @@ We use this version to show the direction and get feedback.
 
 ## Python and Ray image base
 
-There is one Ray on the Golem image. It contains `ray 2.3.1` and `python 3.10.12`.
+There is one Ray on the Golem image. It contains `ray 2.3.1` and `python 3.10.13`.
```

Comment:
```suggestion
There is one Ray on the Golem image. It contains `ray 2.7.1` and `python 3.10.13`.
```

btw, I don't get the "There is one Ray on the Golem image" part... what information does it convey?